.github/workflows/go.yml

```diff
@@ -0,0 +1,24 @@
+name: Go Build
```

Comment:
why not use the Makefile here?

Reply:
I'm not sure if Make is installed in the go action. This is the Go action template provided by Github.

Reply:
Switched to make